RecoBTag/ONNXRuntime/plugins/DeepDoubleXONNXJetTagsProducer.cc

```diff
@@ -189,34 +215,35 @@ void DeepDoubleXONNXJetTagsProducer::make_inputs(unsigned i_jet, const reco::Dee
   start = ptr;
   const auto& tag_info_features = features.tag_info_features;
   *ptr = tag_info_features.jetNTracks;
-  *(++ptr) = tag_info_features.jetNSecondaryVertices;
+  //*(++ptr) = tag_info_features.jetNSecondaryVertices;
```

Comment:
Do all these removed features need to stay here? If so, please add another comment line which explains why they must stay. Otherwise just remove